src/app/features/asset-list/components/stacks-balance-list-item.tsx

```diff
@@ -6,6 +6,6 @@ interface StacksBalanceListItemProps {
   address: string;
 }
 export function StacksBalanceListItem({ address }: StacksBalanceListItemProps) {
-  const balaceDetails = useStxBalance();
-  return <StacksBalanceListItemLayout address={address} {...balaceDetails} />;
+  const balanceDetails = useStxBalance();
+  return <StacksBalanceListItemLayout address={address} {...balanceDetails} />;
```

Comment:
How about refactoring this to merge `StacksBalanceListItem` and `StacksBalanceListItemLayout`? 

`StacksBalanceListItemLayout` calls another layout component `CryptoCurrencyAssetItemLayout` anyway so we are just drilling props down another level

Reply:
I've noticed this too, but it might reveal quite a lot of refactoring overall ...if that is the case I can make a new issue for it, but will try to do here.